models/sipnet/R/model2netcdf.SIPNET.R

```diff
@@ -69,19 +91,13 @@ model2netcdf.SIPNET <- function(outdir, sitelat, sitelon, start_date, end_date,
                                                 ## all "steps", or
                                                 ## if model run doesnt start 
                                                 ## at 00:00:00
-    # get the run dates based on the sipnet output.  we assume that even if the run is partial, the origin is still day 1 of the subset year
-    start_month <- lubridate::month(as.Date(sub.sipnet.output$day[1], origin = paste0(y,"-01-01"))) # gets month
-    ## using this approach to attempt to deal with inconsistent start index 0/1
-    model_start_date <- base::as.Date(ifelse(sub.sipnet.output$day[1]==0,1,sub.sipnet.output$day[1]), origin = paste0(y,"-01-01"))
-    sub_date_range <- seq(model_start_date, by = "day", length.out = length(unique(sub.sipnet.output$day))) ## create new date range in POSIX format
-    # this catches the fact that the number of outputs per day may be different at the end of the year with leap years, if day starts at 1
-    day_repeats <- as.vector(base::table(sub.sipnet.output$day))  
-    # replicate each date based on previously determined day_repeats
-    sub_dates <- rep(sub_date_range,times=day_repeats)  ## expand new date range to match length of model subset subset and steps per day (out.day)
-    jdates <- lubridate::yday(sub_dates)  # create vector of julian days from start to end by year, based on refomatted output dates
+    sub_dates <- as.Date(sipnet2datetime(sub.sipnet.output$day, y, force_cf = FALSE))
```

Comment:
Why `as.Date` here? `cf2datetime` already returns a POSIXct object, which is exactly what `datetime2cf` expects as input. And IIRC, `as.Date` actually drops the time zone information as part of the conversion.

Reply:
whoops.....mistake